Dockerfile

```diff
@@ -11,7 +12,8 @@ COPY /tsconfig.json /usr/src/app/tsconfig.json
 COPY /yarn.lock /usr/src/app/yarn.lock
 
 RUN yarn install \
-  && yarn build
+  && yarn build \
+  && yarn generate-api-client
```

Comment:
maybe in inverse order ? (in case we need the client methods in other typescript sources)